<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pathfinder Unchained: Simple Monster Creation</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Step_1">
       <attribute name="title">
        <string>Step 1: Array</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="4" column="9">
         <widget class="QPushButton" name="continueStep1">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTipDuration">
           <number>4</number>
          </property>
          <property name="text">
           <string>Continue</string>
          </property>
         </widget>
        </item>
        <item row="1" column="9">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="0" colspan="9">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="8">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0" colspan="10">
         <widget class="QTableWidget" name="ArrayTable"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>CR</string>
          </property>
         </widget>
        </item>
        <item row="3" column="8">
         <widget class="QLabel" name="ModLow">
          <property name="text">
           <string>+0</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" colspan="2">
         <widget class="QSpinBox" name="CRSelect">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <number>30</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Array</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QLabel" name="ModHigh">
          <property name="text">
           <string>+0</string>
          </property>
         </widget>
        </item>
        <item row="3" column="5">
         <widget class="QComboBox" name="AbilityMedium"/>
        </item>
        <item row="3" column="2">
         <widget class="QComboBox" name="AbilityHigh"/>
        </item>
        <item row="3" column="7">
         <widget class="QComboBox" name="AbilityLow"/>
        </item>
        <item row="3" column="6">
         <widget class="QLabel" name="ModMedium">
          <property name="text">
           <string>+0</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" colspan="4">
         <widget class="QComboBox" name="ArrayBox"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Step_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Step 2: Grafts</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="Step_3">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Step 3: Spells</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="Step_4">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Step 7: Options</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="Step_5">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Step 8: Skills</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="Step_6">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Step 9: Damage</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPathfinder_Unchained_Simple_Monster_Creation">
    <property name="title">
     <string>Pathfinder Unchained: Simple Monster Creation</string>
    </property>
   </widget>
   <addaction name="menuPathfinder_Unchained_Simple_Monster_Creation"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
